src/_assets/css/_code.scss

```diff
@@ -9,23 +9,20 @@ $pre-bg: #F5F5F5; // TODO(chalin): use $site-color-codeblock-bg?
 
 @import 'code_shared';
 
-// code {
-//   background-color: transparent;
-// }
 pre {
   @include pre-defaults;
 
-  &.prettyprint {
-    .com { color: #6E6E70; }
-    // TODO(chalin): dropping the following overrides would allow us to be consistent with Flutter
-    .kwd { color: #BC0056; }
-    .pln { color: #7500A0; }
-    .lit,
-    .str { color: #00677A; }
-  }
   a {
     font-family: inherit;
     font-weight: inherit;
   }
 }
 
+@media screen {
+  .com { color: #6E6E70; }
+  // TODO(chalin): dropping the following overrides would allow us to be consistent with Flutter
+  .kwd { color: #BC0056; }
+  .pln { color: #7500A0; }
+  .lit,
+  .str { color: #00677A; }
+}
```

Comment:
Note: Prettyprint class overrides have to be done in the same context as the original class definitions (in the vendor CSS), otherwise, the `.highlight` code class `@extends` won't pick up our overrides. Hence this relocation of the code class overrides to be enclosed in `@media screen{ ... }`.